src/material/schematics/ng-add/theming/create-custom-theme.ts

```diff
@@ -12,32 +12,16 @@ export function createCustomTheme(name: string = 'app') {
 // Custom Theming for Angular Material
 // For more information: https://material.angular.io/guide/theming
 @use '@angular/material' as mat;
-// Plus imports for other components in your app.
 
-// Define the theme object.
-$${name}-theme: mat.define-theme((
-  color: (
-    theme-type: light,
-    primary: mat.$azure-palette,
-    tertiary: mat.$blue-palette,
-  ),
-  density: (
-    scale: 0,
-  )
-));
-
-// Include theme styles for core and each component used in your app.
-// Alternatively, you can import and @include the theme mixins for each component
-// that you are using.
-:root {
-  @include mat.all-component-themes($${name}-theme);
-}
-
-// Comment out the line below if you want to use the pre-defined typography utility classes.
-// For more information: https://material.angular.io/guide/typography#using-typography-styles-in-your-application.
-// @include mat.typography-hierarchy($${name}-theme);
```

Comment:
Is there an equivalent for these optional bits with the new system? Or are we changing to a more opinionated stance of either automatically including these as part of the new mixin or just recommending not to use them?

Reply:
I prefer recommending that clients just depend on the system variables where they want them. For example they can make their own custom setup like:

```
h1 {
  font: var(--mat-sys-headline-large);
}

h2 {
  font: var(--mat-sys-headline-medium);
}

h3 {
  font: var(--mat-sys-headline-small);
}
```

My hope is that we can get folks to generally just need to know about `mat.theme` and the available variables. This also seems to match with Material's departure from their previous opinionated typography names: https://m2.material.io/develop/web/guides/typography